src/test/java/com/epam/tc/hw2/BasePageTest.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+public class BasePageTest {
```

Comment:
Why name of this class is BasePageTest? You actually don't have any tests here.

Reply:
It contains setup and tear down methods for subclasses, so I don't know how to rename it. Dmitry Khodakovsky had the same class in the examples without any test methods, and this seemed like a good option to me.